packages/e2e-tests/src/specs/editor/fontCheck/fontCheckMetrics.js

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import {
+  createNewStory,
+  insertStoryTitle,
+  publishStory,
+  addRequestInterception,
+} from '@web-stories-wp/e2e-test-utils';
+
+async function createStoryWithTitle(title) {
+  await createNewStory();
+  await insertStoryTitle(title);
+  await expect(page).toClick('#library-tab-text');
+  const insertButton = await page.waitForXPath(
+    `//button//span[contains(text(), 'Title 1')]`
+  );
+  await insertButton.click();
+  await expect(page).toMatchElement('[data-testid="textFrame"]', {
+    text: 'Title 1',
+  });
+}
+
+async function updateFont(fontFamily) {
+  // Open style pane
+  await expect(page).toClick('li[role="tab"]', { text: /^Style$/ });
+  await expect(page).toClick('button[aria-label="Font family"]');
+  await page.keyboard.type(fontFamily);
+  await page.keyboard.press('ArrowDown');
+  await page.keyboard.press('Enter');
+}
+
+async function addStoryWithFont(title) {
+  await createStoryWithTitle(title);
+  await updateFont('Rock Salt');
+  await publishStory();
+  // Somehow reloading the page after creating the story here helps
+  // with making the test more stable, i.e. toggling dev tools
+  // works more reliably.
+  await page.reload();
+}
+
+function isPlatformMacOS() {
+  return page.evaluate(() => {
+    const { platform } = window.navigator;
+    return platform.includes('Mac') || ['iPad', 'iPhone'].includes(platform);
+  });
+}
+
+async function toggleDevTools() {
+  const areDevToolsOpen = Boolean(await page.$('#web-stories-editor textarea'));
+
+  // Cancel whatever current action to ensure the below shortcut works.
+  await page.keyboard.press('Escape');
+  await page.click('[aria-label="Web Stories Editor"]');
+
+  const isMacOS = await isPlatformMacOS();
+
+  await page.keyboard.down(isMacOS ? 'Meta' : 'Control');
+  await page.keyboard.down('Shift');
+  await page.keyboard.down('Alt');
+  await page.keyboard.down('J');
+  await page.keyboard.up('J');
+  await page.keyboard.up('Alt');
+  await page.keyboard.up('Shift');
+  await page.keyboard.down(isMacOS ? 'Meta' : 'Control');
+
+  // Dev Tools were not open before, but now they should be open.
+  if (!areDevToolsOpen) {
+    await page.waitForSelector('#web-stories-editor textarea');
```

Comment:
This string `#web-stories-editor textarea` is reused a lot. Should it be a const.

Reply:
I don't see a benefit in using a const here, but that's a bit of a personal preference I guess.